test/parallel/test-readline-promises-tab-complete.js

```diff
@@ -114,5 +116,7 @@ if (process.env.TERM === 'dumb') {
     assert.match(output, /^Tab completion error: Error: message/);
     output = '';
   });
-  rli.close();
+  setTimeout(() => {
+    rli.close();
+  });
```

Comment:
IIRC `setTimeout` without a duration used to throw a linting error, did we change it?

Reply:
I've added the delays :slightly_smiling_face: :+1: